docs/doc_4_authentication.md

```diff
@@ -5,3 +5,28 @@ You can [create an API key](https://www.mongodb.com/docs/atlas/configure-api-acc
 
 To learn more about API authentication, see  [Atlas Administration API Authentication](https://www.mongodb.com/docs/atlas/api/api-authentication).
 
+### Use the Atlas Go SDK in Your Code
+
+Construct a new Atlas SDK client, then use the services on the client to
+access different parts of the Atlas Admin API. For example:
+
+```go
```

Comment:
[optional suggestion] I think it might help to add a couple comments in the code example to describe what's happening. Added a couple suggestions below